src/Components/Calendar/Calendar.module.scss

```
@@ -0,0 +1,61 @@
+.calendar {
+  width: 100%;
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: var(--size-6);
+    margin: var(--size-5) var(--size-5) var(--size-3) var(--size-5);
```

아래와 같이 shorthand로도 가능합니다! 저는 이런 식으로 많이 작성을 해서ㅎㅎ

```suggestion
    margin: var(--size-5) var(--size-5) var(--size-3);
```